tests/foreman/ui/test_location.py

```diff
@@ -56,6 +56,14 @@ class LocationTestCase(UITestCase):
     """Implements Location tests in UI"""
     location = None
 
+    @classmethod
+    def setUpClass(cls):
+        """Set up an organization for tests."""
+        super(LocationTestCase, cls).setUpClass()
+        cls.org_ = entities.Organization().search(query={
```

Comment:
Do we need to perform a search for the DEFAULT_ORG instead of using the one with id=1 ?